reproman/support/external_versions.py

```diff
@@ -46,24 +46,29 @@ def __cmp__(self, other):
 
 def _get_annex_version():
     """Return version of available git-annex"""
-    return _runner.run('git annex version --raw'.split())[0]
+    return _runner.run(['git', 'annex', 'version', '--raw'],
+                       expect_fail=True, expect_stderr=True)[0]
```

Comment:
it becomes such a common pattern here (unlike in original datalad version), I wonder if it wouldn't make sense to make a helper here (or may be even at the Runner interface, e.g. `try_run`) e.g.

```python
def _runner_try(cmd, **kwargs):
     return _runner_try(cmd, expect_fail=True, expect_stderr=True, **kwargs)
```
and use it instead of duplicating all the `expect_` kwargs?

Reply:
It (or rather a corrected version that doesn't call itself :) is an overkill in my book, but if you prefer, I'll update it tomorrow.

Reply:
Heh heh ;-) I am not of strong opinion on this, up to you